### PR TITLE
Fixes #23789: system status tab show normal directives rather than system directives

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/23789

Add a parameter to node compliance api to get only system rules

Pass this parameter to api calls of nodecompliance

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala [hunk 1]
```diff
@@ -297,7 +298,8 @@ class ComplianceApi(
       (for {
         level     <- restExtractor.extractComplianceLevel(req.params)
         precision <- restExtractor.extractPercentPrecision(req.params)
-        nodes     <- complianceService.getNodesCompliance()
+        onlySystem = req.params.get("system").flatMap(_.headOption.map(_ == "true")).getOrElse(false)
```
Comment (fanf) [inline webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala:301]: shouldn't that change be removed to not modify public API?

**Review by fanf — COMMENTED** [review]
I think there is a change remaining from early iteration that should not be here. Else, LGTM

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Comment by VinceMacBuche** [comment]
OK, squash merging this PR

**Pull request merged**